core-services/image-mirroring/openshift/mapping_origin_4_16

```diff
@@ -13,4 +13,7 @@ registry.ci.openshift.org/origin/4.16:okd-rpms quay.io/openshift/origin-okd-rpms
 registry.ci.openshift.org/origin/4.16:openstack-installer quay.io/openshift/origin-openstack-installer:4.16 quay.io/openshift/origin-openstack-installer:4.16.0 quay.io/openshift/origin-openstack-installer:latest
 registry.ci.openshift.org/origin/4.16:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.16 quay.io/openshift/origin-operator-marketplace:4.16.0 quay.io/openshift/origin-operator-marketplace:latest
 registry.ci.openshift.org/origin/4.16:ovirt-installer quay.io/openshift/origin-ovirt-installer:4.16 quay.io/openshift/origin-ovirt-installer:4.16.0 quay.io/openshift/origin-ovirt-installer:latest
+registry.ci.openshift.org/origin/4.16:sriov-network-config-daemon quay.io/openshift/origin-sriov-network-config-daemon:4.16 quay.io/openshift/origin-sriov-network-config-daemon:4.16.0 quay.io/openshift/origin-sriov-network-config-daemon:latest
```

Comment:
@jupierce will we see these images mirrored after the postsubmit jobs are executed?